cancermuts_streamlit/streamlit_utils.py

```diff
@@ -0,0 +1,80 @@
+import streamlit as st
+import os
+import pandas as pd
+
+def filter_dataframe(df: pd.DataFrame) -> pd.DataFrame:
+    addFilter = st.checkbox("Add filter")

```

Comment:
I think it would be better to have this line in the main script, so that all the GUI elements are in the main script. Otherwise they are somewhat hidden, for future developers. Also, we can make it so that you just don't use the filtering function unless the checkbox is active. For instance:

```
addFilter = st.checkbox("Add filter")
if addFilter:
    df = filter_dataframe(df)
```

Reply:
So to move the entire function into the main script or to delete the function and just incorporate the code in the main script?

Reply:
Fixed